code/game/machinery/iv_drip.dm

```diff
@@ -121,27 +140,25 @@
 	filling_overlay.color = mix_color_from_reagents(target_reagents.reagent_list)
 	. += filling_overlay
 
-/obj/machinery/iv_drip/MouseDrop(mob/living/target)
+/obj/machinery/iv_drip/MouseDrop(atom/target)
 	. = ..()
-	if(!ishuman(usr) || !usr.canUseTopic(src, be_close = TRUE) || !isliving(target))
+	if(!Adjacent(target) || !usr.Adjacent(target))
```

Comment:
Telekinesis nerf?
```suggestion
	if(!Adjacent(target) || !usr.canUseTopic(src, be_close = TRUE))
```

Reply:
Didn't know telekinesis can work with drag and drop interactions.